src/Form/input-classifiers.test.js

```diff
@@ -147,15 +150,16 @@ describe.each(testCases)('Test $html fields', (testCase) => {
         expectedSubmitFalsePositives = 0,
         expectedSubmitFalseNegatives = 0,
         title = '__test__',
-        hasExtraWrappers = true
+        hasExtraWrappers = true,
+        testContent
     } = testCase
 
     const testTextString = expectedFailures.length > 0
         ? `should contain ${expectedFailures.length} known failure(s): ${JSON.stringify(expectedFailures)}`
         : `should NOT contain failures`
 
-    it(testTextString, () => {
-        const testContent = fs.readFileSync(path.resolve(__dirname, './test-cases', html), 'utf-8')
+    it.concurrent(testTextString, async () => {
```

Comment:
This didn't yield any performance gains for some reason. I've even tried chunking the testCases and running them in parallel, but nothing seemed to matter much 🤷‍♂️. Ran out of time for this scope and the gains are already pretty significant.